app/views/speech/show.html.erb

```diff
@@ -0,0 +1,58 @@
+
+<% content_for :extra_headers do %>
+   <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: @content_item.to_h, schema: :article } %>
+<% end %>
+
```

Comment:
I think you need to include the `<main>` tag here, similar to https://github.com/alphagov/frontend/pull/4225/commits/c501ca1d9ec9c2263ea7c3f1903582bca2333f59#diff-2c4760c884914b406ca821f19956295c799447ffe9a1ae9e780322d1d0c256b4R9

Reply:
I've just seen the tag has been added in a later commit